server/events/output_updater.go

```diff
@@ -53,26 +52,6 @@ type ChecksOutputUpdater struct {
 
 func (c *ChecksOutputUpdater) UpdateOutput(ctx *command.Context, cmd PullCommand, res command.Result) {
 
-	if res.Error != nil || res.Failure != "" {
```

Comment:
Each command runner updates the aggregate status at the end of the operation so removing this block
https://github.com/lyft/atlantis/blob/8cf99abfd260c9da2ab0db5cf9aa940e896de4be/server/events/plan_command_runner.go#L104
https://github.com/lyft/atlantis/blob/8cf99abfd260c9da2ab0db5cf9aa940e896de4be/server/events/apply_command_runner.go#L132
https://github.com/lyft/atlantis/blob/8cf99abfd260c9da2ab0db5cf9aa940e896de4be/server/events/policy_check_command_runner.go#L67
https://github.com/lyft/atlantis/blob/8cf99abfd260c9da2ab0db5cf9aa940e896de4be/server/events/approve_policies_command_runner.go#L77